config/config-example.js

```diff
@@ -133,6 +133,13 @@ exports.crashguardemail = null;
  */
 exports.ssl = null;
 
+/**
+ * Address to bind to.
+ *   This should be kept set to '127.0.0.1' unless you know what you're doing.
+ * @type {string}
+ */
+exports.bindaddress = '127.0.0.1';
```

Comment:
I actually think the right default is `undefined`, corresponding to `0.0.0.0` not `127.0.0.1`

Reply:
The default today is `127.0.0.1`
This would break any current deployment relying on this behaviour.
Plus, since this is a sensitive part of the stack, I'd argue that 127.0.0.1 is much more secure than 0.0.0.0

Reply:
https://nodejs.org/api/net.html#serverlistenport-host-backlog-callback

Reply:
Either way, `undefined` would clearly use the default.

Reply:
So my understanding is that all current deployments are listening on `0.0.0.0`.

Reply:
Okay I tripled checked, and I have *no idea* how I arrived at that wrong conclusion.

My best guess is I must have mixed up testing steps with PM2 (the 6.x.y introduced changes I was not expecting), and you and the NodeJS docs are indeed correct.
I'm very sorry for wasting your time with this false assertion. I'll update my branch with `undefined` as requested.

My PR still stands however, my use case involves binding to a specific IP, not `127.0.0.1` or `0.0.0.0`. Thank you for your time!